src/components/map/CompassComponent.vue

```diff
@@ -17,20 +18,27 @@ onMounted(() => {
 function syncCompass(map: VcsMap) {
   map.postRender.addEventListener(({ map }) => {
     const vp = map.getViewpointSync()
-    if (vp && vp.isValid()) {
+    if (
+      vp &&
+      vp.isValid() &&
+      (previousVp.value === null || !vp.equals(previousVp.value))
+    ) {
       vpPitch.value = vp.pitch
       vpHeading.value = vp.heading
+      previousVp.value = vp
     }
   })
 }
 
 const headingMap = async (event: { angle: number; animate: boolean }) => {
-  console.log('headingMap', event)
   const vp = await rennesApp.maps?.activeMap.getViewpoint()
   if (vp) {
-    vp.heading = event.angle
-    vp.animate = event.animate
-    rennesApp.maps?.activeMap.gotoViewpoint(vp)
+    const vpJson: ViewpointOptions = vp?.toJSON() as ViewpointOptions
+    vpJson.heading = event.angle
+    vpJson.animate = event.animate
+    vpJson.cameraPosition = undefined
```

Comment:
it seems the fix is on this line. I wonder why it was working before without this line.

Reply:
It worked only without tilt. But with tilt, the screen didn't stay at the center.